superset/tasks/schedules.py

```diff
@@ -542,16 +541,15 @@ def schedule_alert_query(  # pylint: disable=unused-argument
     recipients: Optional[str] = None,
 ) -> None:
     model_cls = get_scheduler_model(report_type)
-    dbsession = db.create_scoped_session()
-    schedule = dbsession.query(model_cls).get(schedule_id)
+    schedule = db.session.query(model_cls).get(schedule_id)
```

Comment:
This has me wondering, is there risk that this gets torn down prematurely by the initiating session's closure? My intuition says it shouldn't, but it seems strange that this would have been put here by accident.

Reply:
@villebro I'm unsure though I believe the scoped session is actually the same instance as `db.session` per [here](https://docs.sqlalchemy.org/en/13/orm/contextual.html#contextual-thread-local-sessions), i.e., 

```python
>>> Session = scoped_session(session_factory)
>>> some_session = Session()
>>> some_other_session = Session()
>>> some_session is some_other_session
True
```

Reply:
I threw this into the codebase, and the result was `False`:
```python
>>> db_engine_spec = self.database.db_engine_spec
>>> scoped_session = db.create_scoped_session()
>>> print(scoped_session is db.session)
False
```
Perhaps we should leave these out for now and address them in a follow-up PR?

Reply:
@villebro possibly, though the scoped session is merely running a query as far as I can tell. There’s no complex logic, transaction, etc. being used.

Reply:
@john-bodley good point.